kolibri/plugins/coach/assets/src/vue/coach-exercise-render-page/answer-history/index.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+
+  <div id="answer-history">
+    <h3>{{ $tr('header') }}</h3>
+
+    <ul id="history-list">
+      <template v-for="(attemptLog, index) in reformatedAttemptLogs">
+        <li v-if="index == 0">
+          <p class="item">
+            {{attemptLog.elapse}}
+          </p>
+        </li>
+        <li v-else-if="reformatedAttemptLogs[index -1].elapse != attemptLog.elapse">
+          <p class="item">
+            {{attemptLog.elapse}}
+          </p>
+        </li>
+        <li @click="setSelected(index)" :class="isSeleteced(index)" class="clickable">
+          <div>
+            <mat-svg
+              v-if="attemptLog.hinted"
+              class="item svg-item svg-hint"
+              category="action"
+              name="lightbulb_outline"
+            />
+            <mat-svg
+              v-else-if="attemptLog.correct === 0"
+              class="item svg-item svg-wrong"
+              category="navigation"
+              name="cancel"
+            />
+            <mat-svg
+              v-else
+              class="item svg-item svg-correct"
+              category="action"
+              name="check_circle"
+            />
+            <h3 class="item">
+              {{ $tr('question') }} {{index+1}}
+            </h3>
+          </div>
+        </li>
+      </template>
+    </ul>
+  </div>
+
+</template>
+
+
+<script>
+
+  const today = new Date();
+  const actions = require('../../../actions');
+
+  module.exports = {
+    $trNameSpace: 'CoachExerciseAnswerHistory',
+    $trs: {
+      header: 'Answer History',
+      today: 'Today',
+      yesterday: 'Yesterday',
+      daysAgo: ' Days Ago',
+      question: 'Question',
+    },
+    data: () => ({
+      selectedIndex: 0,
+      reformatedAttemptLogs: [],
+    }),
+    created() {
+      this.attemptLogs.forEach((attemptLog) => {
+        attemptLog.elapse = this.dayElapse(attemptLog.end_timestamp);
+        // use unshift because the original array is in reversed order.
+        this.reformatedAttemptLogs.unshift(attemptLog);
+      });
```

Comment:
Why is `reformatedAttemptLogs` a data attribute initialized inside `created`, instead of a computed property?

If the only difference is that it's reversed, the computed property can just be called `reversedAttemptLogs`

Doing logic like this inside the `created` method and `data` attributes breaks the Vue/Vuex reactivity model.